source/reference/odbc-driver.txt

```diff
@@ -62,7 +62,7 @@ The {+odbc-driver+} is available for:
   .. include:: /includes/temp-warn-odbc-catalina.rst
 
 - Ubuntu 14.04 and 16.04
-- RHEL 7
+- RHEL 8
```

Comment:
No changes have been made regarding RHEL version supported by the ODBC driver. I would not change this because to the best of my knowledge, it is not tested.